### DISCUSSION TITLE
"Wrong" format on command descriptions

### DISCUSSION THREAD

**Discussion by adevade** [filamentphp/filament#6631] [Ideas]
The descriptions for Filament commands does not match Laravel and Livewires conventions, using `Creates` instead of `Create`, and including a period at the end.

![image](https://github.com/filamentphp/filament/assets/1066486/7d64f4db-eff1-4d79-ae7e-c4e67e5c4d48)

---

I've also noticed that some commands are lacking descriptions for arguments and options, possibly making it harder for newcomers to know what does what.

![image](https://github.com/filamentphp/filament/assets/1066486/097c0ebb-e228-4d07-85da-f2a90ba2768a)

---

I can see if I can put in a PR -- if you're open for it -- but I'm not much of a writer 😅 I guess I can fix the formatting at least.

**Comment by danharrin** [comment]
Sure, a PR would be great
